scripts/install.sh

```diff
@@ -1,3 +1,9 @@
 #!/usr/bin/env bash
+
+RED='\033[0;31m'
+NC='\033[0m' # No Color
+echo -e "${RED}[WARNING] This script was deprecated in sc-web 0.8.0.
```

Comment:
```suggestion
echo -e "${YELLOW}[WARNING] This script was deprecated in sc-web 0.8.0.
```
Warning is not error